husky/README.md

```diff
@@ -0,0 +1,110 @@
+Husky
+=====
+
+Common ROS packages for the Clearpath Husky, useable for both simulation and
+real robot operation.
+
+ - husky_control : Control configuration
+ - husky_description : Robot description (URDF)
+ - husky_msgs : Message definitions
+ - husky_navigation : Navigation configurations and demos
+
+For Husky instructions and tutorials, please see [Robots/Husky](http://wiki.ros.org/Robots/Husky).
+
+
+Dependencies/Requirement to build
+=============
+Install ROS2 Humble (follow guide below):
+  - https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html
+
+Install NAV2 and Correlating Dependencies:
+  - sudo apt install ros-humble-slam-toolbox
+  - sudo apt install ros-humble-navigation2 ros-humble-nav2-bringup
+  - sudo apt-get install ros-humble-turtlebot3*
+  - sudo apt install ros-humble-twist-mux
+  - sudo apt install ros-humble-gazebo-ros2-control
+
+Sensor Drivers:
+  - 
+
+Clone this repo and dependending repo's
+```
+cd ~/ros2_ws/src
+git clone https://github.com/TRAILab/husky_trailbot.git
+git clone https://github.com/TRAILab/velodyne.git
+git clone https://github.com/TRAILab/ximea-driver.git
+cd ximea-driver
+./tools/install_sdk.sh
+cd ..
+
+```
+build
+```
```

Comment:
colcon build fails due to pcap.h not found. 

To install all package dependencies, run:
cd ~/trail_ws
rosdep install --from-paths src --ignore-src --rosdistro humble -r -y

To only install libpcap, run:
sudo -H apt-get install -y libpcap0.8-dev